Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM iojs:1.6
+FROM node:4
```

Comment:
That's the issue i'm referring to, if you read it he writes that it's only fixed in npm@2.14.4 (which was not on travis a few weeks ago when i checked, but possibly changed), not sure what version of npm ships with node:4 docker image